app/src/main/java/ch/blinkenlights/android/vanilla/FullPlaybackActivity.java

```diff
@@ -302,10 +302,14 @@ public boolean onCreateOptionsMenu(Menu menu)
 		menu.add(0, MENU_SHARE, 30, R.string.share);
 
 		if (PluginUtils.checkPlugins(this)) {
-			menu.add(0, MENU_PLUGINS, 30, R.string.plugins);
+			menu.add(0, MENU_PLUGINS, 30, R.string.plugins)
+				.setIcon(R.drawable.plugin)
```

Comment:
can you use R.drawable.menu_plugins instead? That's the drawable used in the context menu.

Reply:
Sorry, I can't . I really tried but it's grey and really stands out near the white "Favorite" one.

Reply:
ah right: i forgot that they have a different color.